dashboard/src/util/CommandInstruction.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { Button, Modal, Segment } from 'semantic-ui-react';
+import {CopyToClipboard} from 'react-copy-to-clipboard';
+
+class CommandInstruction extends Component {
+  state = {};
```

Comment:
In this case I'd initialise `state = {open: false}` explicitly because having it `undefined` doesn't really add any meaning distinct from `false`.